### DISCUSSION TITLE
Am I running the latest version of regolith-look-solarized-dark?

### DISCUSSION THREAD

**Discussion by jc00ke** [regolith-linux/regolith-desktop#710] [Q&A] [answered]
tl;dr: I want to include solarized-dark colors into my Xresources file to override a few things but the `color` file doesn't exist on my file system.

Previously, I had overridden my border color to [color_magenta](https://github.com/regolith-linux/regolith-styles/blob/2cf8f01d35d6cfdb75bf7175898927e8798c9fd6/solarized-dark/color#L13) but after upgrading to Regolith 2.1 I've lost my overrides.
![image](https://user-images.githubusercontent.com/18191/186032797-f74cfb31-e982-4180-9705-905853a20eab.png)

```
  ➜ sudo apt remove regolith-look-solarized-dark
  [sudo] password for jesse:
  Reading package lists... Done
  Building dependency tree... Done
  Reading state information... Done
  The following packages were automatically installed and are no longer required:
    arc-icon-theme faba-icon-theme fonts-nerd-font-meslo gruvbox-gtk gtk2-engines-murrine
    moka-icon-theme solarc-theme
  Use 'sudo apt autoremove' to remove them.
  The following packages will be REMOVED:
    regolith-look-solarized-dark
  0 upgraded, 0 newly installed, 1 to remove and 0 not upgraded.
  After this operation, 1,356 kB disk space will be freed.
  Do you want to continue? [Y/n] Y
  (Reading database ... 321882 files and directories currently installed.)
  Removing regolith-look-solarized-dark (0.8.4-1regolith) ...
  ~/.config/regolith2 took 6s

  ➜ sudo apt install regolith-look-solarized-dark
  Reading package lists... Done
  Building dependency tree... Done
  Reading state information... Done
  The following NEW packages will be installed:
    regolith-look-solarized-dark
  0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
  Need to get 1,331 kB of archives.
  After this operation, 1,356 kB of additional disk space will be used.
  Get:1 https://regolith-desktop.org/release-ubuntu-jammy-amd64 jammy/main amd64 regolith-look-solarized-
  dark amd64 0.8.4-1regolith [1,331 kB]
  Fetched 1,331 kB in 0s (3,906 kB/s)
  Selecting previously unselected package regolith-look-solarized-dark.
  (Reading database ... 321872 files and directories currently installed.)
  Preparing to unpack .../regolith-look-solarized-dark_0.8.4-1regolith_amd64.deb ...
  Unpacking regolith-look-solarized-dark (0.8.4-1regolith) ...
  Setting up regolith-look-solarized-dark (0.8.4-1regolith) ...
  ~/.config/regolith2 took 2s

  ➜ regolith-look set solarized-dark
  Executing look loader script: /usr/share/regolith-look/default_loader.sh
  > Updating already existing 'Regolith' profile
  [SUCCESS] Reinstalled Regolith GNOME Terminal version 0.1.0 for the 'Regolith' profile!
  ~/.config/regolith2

  ➜ regolith-look refresh
  /home/jesse/.config/regolith2/Xresources:10:10: fatal error: /usr/share/regolith-look/solarized-dark/co
  lor: No such file or directory
     10 | #include "/usr/share/regolith-look/solarized-dark/color"
        |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  compilation terminated.
  /home/jesse/.config/regolith2/Xresources:10:10: fatal error: /usr/share/regolith-look/solarized-dark/co
  lor: No such file or directory
     10 | #include "/usr/share/regolith-look/solarized-dark/color"
        |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  compilation terminated.
  Executing look loader script: /usr/share/regolith-look/default_loader.sh
  > Updating already existing 'Regolith' profile
  [SUCCESS] Reinstalled Regolith GNOME Terminal version 0.1.0 for the 'Regolith' profile!
  ~/.config/regolith2

  ➜ ls /usr/share/regolith-look/solarized-dark/
  cojdev_github_io-lowpoly.png  gnome-terminal  i3-wm  i3xrocks  ilia.css  root
  ~/.config/regolith2

  ~/.config/regolith2
  ➜ bat /etc/regolith/version
  ───────┬───────────────────────────────────────────────────────────────────────────────────────────────
         │ File: /etc/regolith/version
  ───────┼───────────────────────────────────────────────────────────────────────────────────────────────
     1   │ REGOLITH_VERSION=2.1
     2   │
  ───────┴───────────────────────────────────────────────────────────────────────────────────────────────

  ➜ bat /etc/apt/sources.list.d/regolith.list
  ───────┬───────────────────────────────────────────────────────────────────────────────────────────────
         │ File: /etc/apt/sources.list.d/regolith.list
  ───────┼───────────────────────────────────────────────────────────────────────────────────────────────
     1   │ deb [arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] https://regolith-desktop.org/release-ubuntu-jammy-amd64 jammy main
  ───────┴───────────────────────────────────────────────────────────────────────────────────────────────
```

I feel like my apt sources are pointing to the wrong place after upgrading.

Case in point:

```
  ~/.config/regolith2
  ➜ apt search regolith-look-solarized-light
  Sorting... Done
  Full Text Search... Done
  ~/.config/regolith2
  ➜
```

I would assume that at this point `regolith-look-solarized-light` has been published?

**Comment by kgilmer** [comment, marked as answer]
Hi @jc00ke ,

To override the color of the active window border, the key you seek is `i3-wm.client.focused.color.child_border`.  Here is a screenshot of my machine checking to make sure the change works:

![image](https://user-images.githubusercontent.com/49683/186068923-dd5a7224-350a-45b7-afbc-1db67138beb9.png)

Regarding `regolith-look-solarized-light`, no that look has not been ported from 1.6.  The closest thing is `nevil`.

**Reply by jc00ke** [reply]
Do I have the latest version of `regolith-look-solarized-dark` installed?

Maybe I'm getting confused by the different repos. There's a styles repo and the looks repo IIRC (on phone.) Does the latest dark have the color file?

**Reply by jjsearle** [reply]
this is the current repo used by the looks https://github.com/regolith-linux/regolith-look-extra

I believe the styles repo is obsolete

**Reply by jc00ke** [reply]
Thanks @jjsearle. I'm much less confused now!

I can go back to including the `root` file to get access to the colors but it would feel better if the colors were split out. @kgilmer is that a possible refactor or should I stick with the `root` file?
